internal/client/statistic/monitor.go

```diff
@@ -0,0 +1,127 @@
+package statistic
+
+import (
+	"fmt"
+	"github.com/ilya372317/must-have-metrics/internal/client/sender"
+	"github.com/ilya372317/must-have-metrics/internal/constant"
+	"github.com/ilya372317/must-have-metrics/internal/utils"
+	"runtime"
+	"sync"
+	"time"
+)
+
+const counterName = "PollCount"
+const randomValueName = "RandomValue"
+const minRandomValue = 1
+const maxRandomValue = 50
+
+type Monitor struct {
+	sync.Mutex
+	Data map[string]MonitorValue
+}
+
+func MakeMonitor() Monitor {
```

Comment:
```suggestion
func NewMonitor() Monitor {
```
Конструкторы принято начинать с New, при этом можно и заканчивать, если других конструкторов в пакете нет

Reply:
++